README.md

```diff
@@ -1,2 +1,16 @@
 # PvpBot
 killl your discord friends
+
+
+# Config
+In order to run the bot, you'll need to add your token and prefix to a config file in your Bin directory.
+- Create a folder "bin\Resources"
+- Create a file called "config.json" and place it in the Resources folder
+- Add the following:
+
+```js
```

Comment:
This should be json, not js

Reply:
Good catch, I couldn't remember what the tag was. It shouldn't make a difference though.,